app/economy/models.py

```diff
@@ -134,6 +135,43 @@ def created_human_time(self):
         from django.contrib.humanize.templatetags.humanize import naturaltime
         return naturaltime(self.created_on)
 
+    @property
+    def content_type(self):
+        ct = ContentType.objects.get_for_model(self)
+        return str(ct)
+
+    @property
+    def view_count_redis_key(self):
+        key = f"{self.content_type}_{self.pk}"
+        return key
+
+    def increment_view_count(self):
```

Comment:
@gitcoinco/engineers im consdidering shifting from storing as a total count in redis, to a `ObjectView` django database object, which would allow us to 
1. store time series data
2. also keep track of who views what (useful for segmenting purposes)
3. differentiate index page views (like on /grants) from specific viewings of individual objects (like /grants/35/sustainability fund)

any objection if i do this?  seems like the only counter-point would be DB bloat.

Reply:
There will no doubt be a performance impact in reading and tallying this data - if you want to store that data for later data analysis I think it should be done in parallel with storing the total count in redis. Only the redis count should ever be read from during a web request.

Reply:
good idea.

as a rule, i think we shouldnt write to the DB from most HTTP requests. factoring into a task is almost always a good idea